src/test/java/Tests/AppleTest.java

```diff
@@ -1,4 +1,15 @@
 package Tests;
 
-public class AppleTest {
+import Steps.Steps;
+import org.testng.annotations.Test;
+
+import static Utils.Constants.Constants.APPLE_SEARCHING_PRODUCT_NAME;
+
+public class AppleTest extends BaseTest {
+
+    @Test
+    public void test() {
```

Comment:
The name of test should be readable. I should understand what this test do when I read the name

Reply:
renamed